### ISSUE TITLE
Module descriptor or module name seems to have gone away

### ISSUE THREAD

**Issue by io7m** [marschall/memoryfilesystem#143]
```
$ bnd print memoryfilesystem-2.5.0.jar
[MANIFEST]

Build-Jdk-Spec                          1.8
Bundle-ActivationPolicy                 lazy
Bundle-Description                      An in memory implementation of a JSR-203 file system.
Bundle-License                          https://opensource.org/licenses/MIT
Bundle-ManifestVersion                  2
Bundle-Name                             Memory File System
Bundle-RequiredExecutionEnvironment     JavaSE-1.8
Bundle-SymbolicName                     com.github.marschall.memoryfilesystem;singleton:=true
Bundle-Version                          2.5.0
Created-By                              Maven JAR Plugin 3.3.0
Export-Package                          com.github.marschall.memoryfilesystem;version="2.5.0"
Import-Package                          javax.annotation;resolution:=optional
Manifest-Version                        1.0
Require-Capability                      osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"


[IMPEXP]

Import-Package
  javax.annotation                       {resolution:=optional}

Export-Package
  com.github.marschall.memoryfilesystem  {version=2.5.0}
```

There's no `Automatic-Module-Name`, and:

```
$ unzip -l memoryfilesystem-2.5.0.jar | grep module
$
```

... no `module-info.class`.

**Comment by io7m** [comment]
Version `2.4.0` did have an explicit module descriptor, so I'm not sure what happened.

**Comment by marschall** [comment]
No idea yet why it is missing from the release JAR. When I build it locally it is present.

**Comment by marschall** [comment]
I released 2.5.1 which at least locally has a `module-info.class`

**Comment by io7m** [comment]
Thanks! That appears to have fixed it. There's a `module-info.class` there now.
